specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2025-08-02-preview/managedClusters.json

```diff
@@ -8676,6 +8677,10 @@
             "value": "AzureLinux3",
             "description": "Use AzureLinux3 as the OS for node images. Azure Linux is a container-optimized Linux distro built by Microsoft, visit https://aka.ms/azurelinux for more information. For limitations, visit https://aka.ms/aks/node-images. For OS migration guidance, see https://aka.ms/aks/upgrade-os-version."
           },
+          {
+            "value": "Flatcar",
+            "description": "Use Flatcar as the OS for node images. Flatcar is a container-optimized Linux distro built by Kinvolk, visit https://www.flatcar-linux.org/ for more information."
```

Comment:
@jepio what description we want to have here? I went with what autopilot generated. Maybe we can mention CNCF distro here?